lib/vault.py

```diff
@@ -186,15 +186,16 @@ def configure(self):
             provisioner_policies.append(name)
 
         # AWS Authentication Backend
-        #Enable AWS auth in Vault
-        if 'aws' not in client.list_auth_backends():
-            try:
-                client.enable_auth_backend('aws')
-            except Exception as e:
-                print("Error while enabling auth back end: " + e)
-                sys.exit(1)
-        else:
-            print("aws auth backend already created.")
+        # Enable AWS auth in Vault
+        try:
```

Comment:
Curious as to why the check to see if the backend was already enabled was removed. Just a little concerned that the AWS auth backend and secret backend have different logic to enable them (meaning that we may be missing something in one of the blocks of code).

Reply:
Hey Derek, I am just checking for it differently. 
If the backend was already enabled the path is reported to already be in use by hvac, So I just catch the exception, check to see if the message i s "path is already in use", and if so I pass the check. 
For some reason even after being enabled, the aws backend wasn't showing up on the output of
`        if 'aws' not in client.list_auth_backends():`

Reply:
Is there a more explicit exception that you can use?

Reply:
Nevermind, I see you're checking the exception message.

Reply:
It looks like vault now returns data with the mount point as the key, so `aws/` instead of `aws`. You should be able to keep the original `if 'aws/' not in client.list_` check.

I also triggered the same error lower down with `client.list_secret_backends()` as well, so the error handling logic should be the same.

Reply:
Alright, I made the update and we are back in business!